requirements.txt

```diff
@@ -1,7 +1,8 @@
-django==4.1
+django==5.1.3
 flake8==5.0.4
 flake8-quotes==3.3.1
 flake8-variables-names==0.0.5
 pep8-naming==0.13.2
 django-debug-toolbar==3.2.4
-django-crispy-forms==1.14.0
+crispy-bootstrap4==2024.10
+django-crispy-forms==2.3
```

Comment:
requirements.txt

django==4.1
flake8==5.0.4
flake8-quotes==3.3.1
flake8-variables-names==0.0.5
pep8-naming==0.13.2
django-debug-toolbar==3.2.4
django-crispy-forms==1.14.0
crispy-bootstrap4 <<<<- -----------------------------------------------pay attention


try this commands:
1. pip uninstall django
2. pip uninstall django-crispy-forms
3. pip install -r requirements.txt